Project2/src/main/webapp/resources/features/login.html

```diff
@@ -1,6 +1,4 @@
-<h1>Log into JAB'R</h1>
-<div class="alert alert-danger" ng-if="errorMessage">{{errorMessage}}</div> <br>
-
-Username: <input data-ng-model="username" type="text" placeholder="username">
-Password: <input data-ng-model="password" type="password" placeholder="password">
+<h1>JABR</h1>
```

Comment:
Are we taking the error message out? Or was this part of the slightly older version?